### PR TITLE
dev: hive readme

### PR DESCRIPTION
**Pull request by greged93**
Time spent on this PR: 0.5 day

Resolves: #871 

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [x] Documentation content changes
- [ ] Testing

# What is the new behavior?
- Adds details about all skipped hive tests in the README + details about the skipped fields for passing tests.
- Boy scout commit to simplify `fee history`.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,14 +226,112 @@ You can take a look at `rpc-call-examples` directory. Please note the following:
   with an updated nonce using the
   [provided python script](https://github.com/sayajin-labs/kakarot/blob/main/scripts/utils/kakarot.py#L273).
 
+## Testing
+
+### Hive
+
+The [Hive] end-to-end test suite is set up in the Github Continuous Integration (CI)
```
Comment (Eikix) [inline README.md:233]: link to hive?

File: README.md [hunk 2]
```diff
@@ -226,14 +226,112 @@ You can take a look at `rpc-call-examples` directory. Please note the following:
...
+flow of the repository. This ensures a safe guard when modifying the current RPC
+implementation and/or the [execution layer](https://github.com/kkrt-labs/kakarot).
+
+Due to the current existing differences between the Kakarot EVM implementation which
+aims to be a type 2 ZK-EVM (see the blog post from [Vitalik](https://vitalik.eth.limo/general/2022/08/04/zkevm.html)
+for more details), some of the Hive tests need to be skipped or slightly modified
+in order to pass.
+
+For the [hive rpc tests](https://github.com/kkrt-labs/hive/tree/master/simulators/ethereum/rpc),
+all the websocket related tests are skipped as websocket aren't currently supported
```
Comment (Eikix) [inline README.md:243]:
```suggestion
all the websockets related tests are skipped as websockets aren't currently supported
```

Comment (greged93) [reply README.md:243]: I wrote `websockets` but trunk was telling me the word doesn't exist :/

**Review by Eikix — DISMISSED** [review]
lgtm

**Review by Eikix — APPROVED** [review]
lgtm

**Pull request merged**